apps/vyper/README.md

```diff
@@ -0,0 +1,37 @@
+# vyper-remix
+Vyper Plugin for Remix IDE.
+
+
+## How to get started
+### Remote plugin
+This plugin is hosted at https://vyper.remixproject.org .
+To use it, open the plugin manager in Remix and click the `Vyper` button in the plugin section.
+
+### Local plugin
+You can host this plugin in your local environment.
```

Comment:
May be you should add that clone this repo and set it as your working directory before running those commands